### DISCUSSION TITLE
Windows 10 Receivers

### DISCUSSION THREAD

**Discussion by esdad697** [svpcom/wfb-ng#178] [Questions]
FAQ says, "Wifibroadcast is not tied to any GPU - it operates with UDP packets. But to get RTP stream you need a video encoder (with encode raw data from camera to x264 stream)."

Are there any solutions for receiving video under Windows 10, preferably in Python?

**Comment by svpcom** [comment]
You can use gstreamer
https://gstreamer.freedesktop.org/documentation/installing/on-windows.html?gi-language=c
